sentry-samples/sentry-samples-android/build.gradle.kts

```diff
@@ -133,5 +133,5 @@ dependencies {
     implementation(Config.Libs.composeNavigation)
     implementation(Config.Libs.composeMaterial)
 
-    debugImplementation(Config.Libs.leakCanary)
```

Comment:
Any specific reason you commented this out?

Reply:
nope, reverted